pkg/metrics/exporter/export.go

```diff
@@ -69,6 +88,17 @@ func (e *Exporter) ExportFsMetrics(m *types.FsMetrics, imageRef string) error {
 	return e.output()
 }
 
+func ExportNydusdEventMetric(daemonID string, event string) error {
+	e, err := getExporter()
+	if err != nil {
+		return err
+	}
+
+	daemon.NydusdEvent.WithLabelValues(daemonID, time.Now().Format("2006-01-02 15:04:05.000"), event).Inc()
+
+	return e.output()
```

Comment:
It's a bad idea to write metrics to files each time this function is called. It will harm snapshotter performance.
I suppose only write them out when querying metrics via HTTP api

Reply:
Done.